Remotion/Web/Core/UI/Controls/CommandInfo.cs

```diff
@@ -157,7 +158,7 @@ private void AddDiagnosticMetadataAttributes (HtmlTextWriter writer)
 
     private bool IsTriggeringNavigation ()
     {
-      return _href != "#";
+      return _href != SafeServiceLocator.Current.GetInstance<IFallbackNavigationUrlProvider>().GetURL();
```

Comment:
I think this is breaking the intended semantics. It was probably hacky from the start but it worked due to lack of creative usage.

New semantics:
IsTriggeringNavigation tests
1. _href being not null and starting with "`#`"
2. _onClick being null or empty or not contains "`__doPostBack`"

IsTriggeringPostBack tests just for _onClick contains "__doPostback"

We assume that the developer correctly implements "__doPostback" with a "return false". This is a reasonable assumption when the intention is simply to detect the intended behavior, not to find / compensate for bugs or strange implementations. This may break the webtests. we need to check the result and and act if it does.

Reply:
We do not quite understand what is meant here. Will ask about it in standup :)

Reply:
We now run into the problem of the tests not being green. We should probably have a quick talk about how to fix this.

Reply:
@Frazerus discussed this about the general direction of the problem with @KuJo7. I'm sure you two will talk this through tomorrow :)

I'll do a final review/hint if there's still unresolved stuff.